src/items.rs

```diff
@@ -607,6 +607,112 @@ pub fn format_struct(context: &RewriteContext,
     }
 }
 
+pub fn format_trait(context: &RewriteContext, item: &ast::Item, offset: Indent) -> Option<String> {
+    if let ast::ItemKind::Trait(unsafety, ref generics, ref type_param_bounds, ref trait_items) =
+           item.node {
+        let mut result = String::new();
+        let header = format!("{}{}trait {}",
+                             format_visibility(item.vis),
+                             format_unsafety(unsafety),
+                             item.ident);
+
+        result.push_str(&header);
+
+        // TODO: Add max_width checking
```

Comment:
Looks like it can be removed, I did add max width checking. Looks like I forgot to remove that comment.